src/ansys/edb/core/session.py

```diff
@@ -553,3 +555,23 @@ def get_variable_server_stub():
     VariableServerServiceStub
     """
     return StubAccessor(StubType.variable_server).__get__()
+
+
+def has_session(port_num):
```

Comment:
minor: can you make it a private method?

also raising an error in bool function is a little odd.

maybe change to `ensure_session` or something and bring the lines 509-510 inside as well